lib/dynamodb_framework/dynamodb_table_manager.rb

```diff
@@ -116,6 +116,25 @@ def update_index_throughput(table_name, index_name, read_capacity, write_capacit
       DynamoDbFramework.logger.info "[#{self.class}] -Table: [#{table_name}] updated."
     end
 
+    def update_ttl_attribute(table_name, time_to_live_status, attribute_name)
```

Comment:
`enabled` rather than `time_to_live_status`? wdyt?

Reply:
Perfectly sensible, `time_to_live_status` was a leftover from when I was trying to use a different dynamodb client method I'll update it.